### PR TITLE
Customize bullets

### PR DESCRIPTION
**Pull request by gogumai**
Give the user the possibility yo customise the bullets and the
container that contains them.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -196,12 +199,18 @@ export default class Carousel extends Component {
...
+            style={i === this.state.currentPage ?
+              [styles.chosenBullet, this.props.chosenBulletStyle] :
+              [styles.bullet, this.props.bulletStyle]}
+          />
         </TouchableWithoutFeedback>);
     }
     return (
       <View style={styles.bullets}>
-        {bullets}
+        <View style={[styles.bulletsContainer, this.props.bulletsContainerStyle]}>
```
Comment (phil-r) [inline index.js]: Is there any reason why you added extra `View` ?

Comment (gogumai) [inline index.js]: So we can maintain the absolute position, margins, and centering with the outer `View` and only customize  the "real"  bulletsContainer (inner `View`).

But if you see a better or more elegant aproach, I'll be glad to know about it :)

**Comment by razvan-tudosa** [comment]
Are you guys going to do something about this PR?

**Comment by phil-r** [comment]
@razvan-tudosa Thanks for raising this up, I'll merge and release new version

**Pull request merged**